### PR TITLE
Features: Add signup flow related features for Newsletter and Link in Bio

### LINKED ISSUE #67197: Customize plans page for newsletter and link in bio

### PR DESCRIPTION
**Pull request by heavyweight**
### Context for translation

| Newsletter Plans  | Link in Bio Plans |
| ------------- | ------------- |
|  Screenshot 2022-08-26 at 18 59 17 | Screenshot 2022-08-26 at 18 58 44  |
| Screenshot 2022-08-30 at 13 46 30 | Screenshot 2022-08-30 at 13 47 02 |

#### Proposed Changes

* We need to customize the plans page based on the user's signup flow.
* For Newsletter and Link in Bio we need to show custom features, for both desktop and mobile devices.

#### Testing Instructions
- Checkout this branch
- Run `yarn start`
- Checkout the desktop and mobile versions of the plans page by following the [newsletter](http://calypso.localhost:3000/setup/intro?flow=newsletter) and [link-in-bio](http://calypso.localhost:3000/setup/intro?flow=link-in-bio) flow

### CODE REVIEW

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7498000

Thank you @heavyweight for including a screenshot in the description! This is really helpful for our translators.

**Review by renancarvalho — APPROVED** [review]
Worked fine! Tested tailored flows and `/start/domain/domain-only`
Good job! 🎖️

File: packages/calypso-products/src/plans-list.tsx [hunk 1]
```diff
@@ -674,6 +733,63 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
...
+		FEATURE_LIVE_CHAT_SUPPORT,
+		FEATURE_REAL_TIME_ANALYTICS,
+		FEATURE_PREMIUM_THEMES,
+	],
+	getLinkInBioDescription: () =>
+		i18n.translate(
+			'Take your site further, faster. Get everything included in Personal, ' +
+				'plus premium design themes, baked-in video uploads, ad monetization, deep visitor ' +
+				'insights from Google Analytics, and live chat support.'
+		),
```
Comment (alshakero) [inline packages/calypso-products/src/plans-list.tsx]: this `+` might cause translation issues. Let's keep it one string.

**Review by alshakero — CHANGES_REQUESTED** [review]
Code looks good. I have a hunch that it's over-complicated, but it works great in the tailored and normal flows, so let's ship (code-wise). Just noting some outdated copy.

**Newsletter:** 
This phrase: https://www.figma.com/file/BcUdhLlhAp7UHtQHNTI4zA/v1.-Tailored-Onboarding-%26-Launchpad?node-id=4495%3A210304

**LiB:**
https://www.figma.com/file/BcUdhLlhAp7UHtQHNTI4zA/v1.-Tailored-Onboarding-%26-Launchpad?node-id=4174%3A148599

**Review by alshakero — APPROVED** [review]
Tested and regession-tested and looks great. 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7498000

Thank you @heavyweight for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7498000).

**Pull request merged**